docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1316,43 +1317,57 @@ See the proofs for this approach below.
-                           child started eating iron-fortified food by the end of the simulation
-  mother_iron_group     := population subgroup indicating in which scenario (if any) the
-                           child's mother got sufficient iron-fortification during pregnancy
-
-  # Population level parameters
-  hb_shift              = effect size on hemoglobin (normally disributed, mean 3.0 g/L)
-  bw_response           = dose-response on birthweight (normally distributed, mean 1.51 g/mg)
-  baseline_coverage(t)  = iron fortification coverage at time t in baseline scenario
+  iron_coverage_group       := population subgroup indicating in which scenario (if any) the child's
+                               household received iron-fortified food by the end of the simulation
+  mother_iron_group         := population subgroup indicating in which scenario (if any) the child's
+                               mother got sufficient iron-fortification during pregnancy
+
+  ## Population level parameters:
+  hb_shift                = effect size on hemoglobin (normally disributed, mean 3.0 g/L)
+  bw_response             = dose-response on birthweight (normally distributed, mean 1.51 g/mg)
+  baseline_coverage(t)    = iron fortification coverage at time t in baseline scenario
     # Note: If delta_coverage is defined to be 0 in the baseline scenario, then
     # the below strategies should work in both baseline and intervention scenarios
-  delta_coverage(t)     = change in coverage from baseline
-  coverage(t)           = iron fortification coverage at time t
-                        = baseline_coverage(t) + delta_coverage(t)
+  delta_coverage(t)       = change in coverage from baseline
+  coverage(t)             = iron fortification coverage at time t
+                          = baseline_coverage(t) + delta_coverage(t)
     # Multiplier on hb_shift due to age
-  hb_age_fraction(age)  = 0 if age<0.5 else (age-0.5)/1.5 if age<2 else 1
+  hb_age_fraction(age)    = 0 if age<0.5 else (age-0.5)/1.5 if age<2 else 1
     # Multiplier on hb_shift due to lag in response time
-    # (value is irrelevant if simulant hasn't yet received fortification)
-  hb_lag_fraction(time_since_coverage)
-                        = time_since_coverage/0.5 if 0<=time_since_coverage<0.5 else 1
-  iron_concentration    = concentration X of iron in the country's fortified flour
-
-  # Individual level attributes
-  p_i                     = propensity of simulant and mother for exposure to unfortified flour (uniform(0,1))
-  daily_flour_i           = average daily flour consumption Y of simulant's mother (sampled from population distribution)
+    # Negative time_since_fortified indicates child has not yet started eating fortified food
+  hb_lag_fraction(time_since_fortified)
+                          = (0 if time_since_fortified < 0 else
+                             time_since_fortified/0.5 if time_since_fortified < 0.5 else
+                             1)
```

Comment:
The `0 if time_since_fortified < 0` portion is intended to apply to those who are not covered, right?

Reply:
Yes, if `time_since_fortified < 0`, then either they'll never get coverage, or they haven't gotten it yet.

It doesn't actually matter whether this gets assigned to 0 here or not, because it gets multiplied by other things that will be 0, but I think assigning it to 0 makes the most sense for uncovered simulants.

Reply:
Cool, that sounds good